.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+**Reason for Change**:
+<!-- What does this PR improve or fix in Kaito? Why is it needed? -->
+
+**Requirements**
+
+- [ ] squashed commits
+- [ ] included documentation
+- [ ] added unit tests and e2e tests (if applicable).
+
+**Issue Fixed**:
+<!-- If this PR fixes GitHub issue 4321, add "Fixes #4321" to the next line. -->
+
+**Please answer the following questions with yes/no**:
```

Comment:
We can exclude this question for now. It is better to start with a simple template.